common/known_keys.go

```diff
@@ -0,0 +1,9 @@
+package common
+
+var (
```

Comment:
I don't think these need to be in common. Related to my comment about storing the bootnodes -- if we provide functions instead of looking up keys in the DB, this would only be needed in the trie package

Reply:
changed to provide store/load genesis data functions